com.stremio.Service.json

```diff
@@ -0,0 +1,54 @@
+{
+    "app-id": "com.stremio.Service",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "44",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "stremio-service",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--talk-name=org.kde.StatusNotifierWatcher"
+    ],
+    "build-options": {
+        "append-path": "/usr/lib/sdk/rust-stable/bin"
+    },
+    "modules": [{
+        "name": "service",
+        "buildsystem": "simple",
+        "build-options": {
+            "env": {
+                "CARGO_HOME": "/run/build/service/cargo"
+            }
+        },
+        "build-commands": [
+            "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+            "cargo --offline build --release --features=bundled,offline-build --verbose"
+        ],
+        "post-install": [
+            "install -Dm755 server.js -t /app/bin/stremio-service/",
+            "install -Dm755 ./target/release/stremio-service -t /app/bin/stremio-service/",
```

Comment:
this make `stremio-service` no found what is found from `$PATH` directory is the directory with that name.

I always frown when I see directories copied into the `bin` directory

Reply:
That's why we were using a path in the command property 
The previous build worked

Reply:
the command should through `PATH`. It is not.

Reply:
What does this mean ?
Can you provide a clear explanation as to what to do to resolve this ?

Reply:
the binary should be in `/app/bin`. Or a wrapper script to call it. And that wrapper should be the name of the executable is both `command` and the `.desktop` file.

Reply:
It has been updated to use a wrapper